stacks/product-local/main.tf

```diff
@@ -35,7 +35,7 @@ provider "kubernetes" {
 }
 
 provider "helm" {
```

Comment:
do we still need all 3 helm providers?

Reply:
Not the way we are using the products now. If we wanted to be able to have the products deploy to other clusters we would but we should have to add a bunch of other stuff anyway.

Reply:
gotcha.  i'm fine keeping them around then